src/domain/core/src/services/verification_service.rs

```diff
@@ -140,6 +146,10 @@ pub trait VerificationMultiListener: Send + Sync {
     fn begin_verify(&self, _dataset: &DatasetHandle) -> Option<Arc<dyn VerificationListener>> {
         None
     }
+
+    fn begin_multi_verify(&self) -> Option<Arc<dyn VerificationMultiListener>> {
+        None
```

Comment:
What is the point of this method?

`begin_verify` is a way to convert from "multi-listener" into a "single-listener".

This new method returns "multi-listener" from what is already a "multi-listener"...

We don't have such a method in any of the other listener interfaces.